### DISCUSSION TITLE
Associate wine with recipes

### DISCUSSION THREAD

**Discussion by pecbeck** [mealie-recipes/mealie#3558] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Add feature to include / recommend wine to match meals. Ability to add wine, image and notes etc, and associate with recipes

### Please provide a concise description of the feature that would resolve your issue.

Add feature to include / recommend wine to match meals. Ability to add wine, image and notes etc, and associate with recipes

### Please consider and list out some caveats or tradeoffs made in your design decision

None

### Additional Information

- [X] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by hjk321** [comment]
For what it's worth, this can pretty much be accomplished already by adding a note directly to the recipe. What advantage is gained by separating this to its own system?
